modules/telemetry/telemetry.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace VIPWorkflow\Modules\Telemetry;
+
+require_once __DIR__ . '/events/status-events.php';
+require_once __DIR__ . '/events/notification-events.php';
+require_once __DIR__ . '/events/settings-events.php';
+
+use Automattic\VIP\Telemetry\Tracks;
+use VIPWorkflow\Modules\Telemetry\Events\Status_Events;
+use VIPWorkflow\Modules\Telemetry\Events\Notification_Events;
+use VIPWorkflow\Modules\Telemetry\Events\Settings_Events;
+
+class Telemetry {
+	/**
+	 * Tracker instance
+	 *
+	 * @var Tracks
+	 */
+	protected Tracks $tracks;
+
+	/**
+	 * Constructor
+	 */
+	public function __construct() {
+		$this->tracks = new Tracks( 'vip_workflow_' );
+	}
+
+	/**
+	 * Initialize the module and register event callbacks
+	 */
+	public function init(): void {
+		// Add custom status events
+		$status_events = new Status_Events( $this->tracks );
+		add_action(
```

Comment:
Minor: I like that you've split the code up to gather events into separate classes, but it could be even better separated if we put the action registration calls in those classes as well, as `Status_Events`' functions are tightly bound to the action registration. You don't need to do this if you disagree.

Reply:
That's a good idea actually. Changed in a0f9415fec276b3acf490773e9dc9db2548f8d36

Reply:
Thank you, looks great!